crates/fluvio-spu-schema/src/server/consumer_offset.rs

```diff
@@ -100,3 +102,60 @@ impl ConsumerOffset {
         }
     }
 }
+
+#[derive(Decoder, Encoder, Default, Debug)]
+pub struct GetConsumerOffsetRequest {
+    pub replica_id: ReplicaKey,
```

Comment:
to follow other request conventions, this should be list of request.
`replica_id` should be optional

Reply:
We have a `FetchConsumerOffsetsRequest` already that does it, maybe merge them? 
also make `replica_id` and `consumer_id` optional

Reply:
sure if that make sense

Reply:
sure, done!